src/SFA.DAS.AANHub.Application/Members/Queries/GetMemberActivities/GetMemberActivitiesQueryValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+
+namespace SFA.DAS.AANHub.Application.Members.Queries.GetMemberActivities;
+public class GetMemberActivitiesQueryValidator : AbstractValidator<GetMemberActivitiesQuery>
+{
+    public const string MemberIdMissingMessage = "MemberId must have a value";
+
+    public GetMemberActivitiesQueryValidator()
```

Comment:
There is already a MemberId validator which should be reused here. https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/main/src/SFA.DAS.AANHub.Application/Common/Validators/MemberId/MemberIdValidator.cs

Reply:
Done